site/pages/gcweb-theme/static-header-footer/bootstrap-3.html

```diff
@@ -12,7 +12,7 @@
 	<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/css/theme.min.css">
 	<noscript><link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/wet-boew/css/noscript.min.css" /></noscript>
 </head>
-<body vocab="http://schema.org/" typeof="WebPage">
+<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
```

Comment:
Missing a resource 
* at line 31 for the publisher info 
* at line 88 for the main element.